aspnetcore/fundamentals/http-requests.md

```diff
@@ -209,36 +209,40 @@ An approach to managing regularly used policies is to define them once and regis
 
 [!code-csharp[Main](http-requests/samples/Startup.cs?name=snippet10)]
 
-In the preceding code, a PolicyRegistry is added to the `ServiceCollection` and two policies are registered with it. In order to use a policy from the registry, the `AddPolicyHandlerFromRegistry` method is used, passing the name of the policy to apply.
+In the preceding code, a `PolicyRegistry` is added to the `ServiceCollection` and two policies are registered with it. To use a policy from the registry, the `AddPolicyHandlerFromRegistry` method is used, passing the name of the policy to apply.
```

Comment:
Comma before "and" in the first sentence. However, this could also go ...

> In the preceding code, two policies are registered when the \`PolicyRegistry\` is added to the \`ServiceCollection\`.